### ISSUE TITLE
point of getting_started.rst?

### ISSUE THREAD

**Issue by tom-osika** [johnwparent/APBPoster#10]
I assume that this page will be integrated into the final poster so that it contextually makes sense? https://github.com/johnwparent/APBPoster/blob/master/getting_started.rst

Right now its just a standalone rst file

**Comment by johnwparent** [comment]
Yeah, this is meant to point to the binder instance with some light instructions and a brief intro.

**Comment by tom-osika** [comment]
> Yeah, this is meant to point to the binder instance with some light instructions and a brief intro.

I assume the same goes for the last two lines here? https://github.com/johnwparent/APBPoster/blame/master/index.rst
